### PR TITLE
Add char count and max char info near textarea of comments area

### PR DESCRIPTION
**Pull request by danielcaldas**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

https://github.com/lbryio/lbry-desktop/issues/2859

## What is the current behavior?
No information about character count. Also no limit on the textarea for comments.

## What is the new behavior?
The new behavior is demonstrated in the below GIF.

counter-info

### CODE REVIEW

File: src/ui/scss/component/_comments.scss [hunk 1]
```diff
@@ -45,3 +45,8 @@
   flex-basis: 200px;
   text-align: right;
 }
+
+.comment__char-count {
+  align-self: flex-end;
+  font-size: 12px;
```
Comment (seanyesmunt) [inline src/ui/scss/component/_comments.scss]: Can you change this to `font-size: var(--font-label);`

File: src/ui/component/common/form-components/form-field.jsx [hunk 2]
```diff
@@ -141,10 +146,15 @@ export class FormField extends React.PureComponent<Props> {
           </div>
         );
       } else if (type === 'textarea') {
+        const hasCharCount = charCount !== undefined && charCount >= 0;
+        const countInfo = hasCharCount && (
+          <span className="comment__char-count">{`${charCount || ''}/${textAreaMaxLength}`}</span>
```
Comment (seanyesmunt) [inline src/ui/component/common/form-components/form-field.jsx]: Instead of an empty string, this should just be `0`

Comment (danielcaldas) [reply src/ui/component/common/form-components/form-field.jsx]: Oh sure, missed that one.

File: src/ui/scss/component/_comments.scss (hunk 1 above)
Comment (danielcaldas) [reply src/ui/scss/component/_comments.scss]: Will do

**Comment by tzarebczan** [comment]
@danielcaldas for the PR! Can we show you some [appreciation](https://lbry.com/faq/appreciation)? 

Make sure to check out https://lbry.tech/contribute and sign up for the dev mailing list at the bottom! 

Also, see https://lbry.com/news/hacktoberfest-2019 for the latest Hacktoberfest details.

**Pull request merged**